Apps/W1/HybridGP/app/src/Migration/Accounts/GPAccountMigrator.codeunit.al

```diff
@@ -74,6 +75,65 @@ codeunit 4017 "GP Account Migrator"
         Sender.ModifyGLAccount(true);
     end;
 
+    [EventSubscriber(ObjectType::Codeunit, Codeunit::"GL Acc. Data Migration Facade", 'OnCreateOpeningBalanceTrx', '', true, true)]
+    procedure OnCreateOpeningBalanceTrx(VAR Sender: Codeunit "GL Acc. Data Migration Facade"; RecordIdToMigrate: RecordId)
+    var
+        GPAccount: Record "GP Account";
+    begin
+        if RecordIdToMigrate.TableNo() <> Database::"GP Account" then
+            exit;
+
+        GPAccount.Get(RecordIdToMigrate);
```

Comment:
Is it possible for the record not to exist? Should we stop the code execution if the record does not exist?

Reply:
it should not. The data migration facade handles this. It is the same pattern for customers, vendors, items.